collector/run_production.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+if [[ -z $CASSANDRA_NODES ]]; then
+  echo "** ERROR: You need to specify CASSANDRA_NODES env var."
+  echo "CASSANDRA_NODES should contain a comma separated list of Cassandra node hosts"
+  exit 1
+fi
+
+SERVICE_NAME="zipkin-collector-service"
+CONFIG="${SERVICE_NAME}/config/collector-cassandra.scala"
+
+hosts=$(echo $CASSANDRA_NODES | tr "," "\n")
+
+CASSANDRA_NODES_CFG_STRING=""
```

Comment:
this part is confusing me.. it looks like we are splitting a comma-separated string and joining it again. Am I missing something?